code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -144,6 +144,9 @@ proc/issyndicate(mob/living/M as mob)
 	var/mob/living/carbon/human/M = synd_mind.current
 	var/obj/item/organ/external/head/head_organ = M.get_organ("head")
 
+	M.mutations = list()
+	M.disabilities = 0
+	M.overeatduration = 0

```

Comment:
I had another look, and I think this is the most global solution without having to manually remove every disability manually with their procs. This also gives a message, but I don't think it's worth implementing a silent var just for this. 

If at some point we get datum antags this should probably become a generalized proc for randomizing antags.